src/components/navbar/Create.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { CreateStyleWrapper } from "./Styles/CreateStyleWrapper";
+
+const Create = () => {
+  const [title, setTitle] = useState("");
+  const [director, setDirector] = useState("");
+  const [overview, setOverview] = useState("");
+  const [rating, setRating] = useState("");
+  const [img, setImg] = useState(
+    "https://i.ytimg.com/vi/VzvdoLeXClg/movieposter_en.jpg"
+  );
+  const navigate = useNavigate();
+
+  const handleAddMovieSubmit = (e) => {
+    e.preventDefault();
+    const movie = { title, director, overview, rating, img };
+    console.log(movie);
+    fetch("http://localhost:8001/movies/", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(movie),
+    }).then(() => {
```

Comment:
dont move this to the api file, redirection should be handled here

Reply:
Done